### DISCUSSION TITLE
opacity of editor elements

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21999] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/78441708/210190597-5dca460f-2bba-49b7-87eb-b46d2981eb81.png)
![image](https://user-images.githubusercontent.com/78441708/210190599-b8471759-76a3-4b15-aab3-8e6904125d15.png)

In the screenshot from stable, the elements (the slider combo'd 1)  are visible through the timeline; in Lazer, this is not so. 
Probably all the elements around the playfield should be translucent?
or we could add a similar feature to background dim in the editor, with a few options for mappers.

is it also possible this will come as Lazer skinning becomes more complete?

**Comment by peppy** [comment, marked as answer]
I think if anything, we will have an interface toggle shortcut which will hide everything. And potentially some kind of auto-hide (ie. dim  automatically when the mouse cursor is in the playfield).
